src/main/java/echoquery/utils/VisualizationUtil.java

```diff
@@ -0,0 +1,41 @@
+package echoquery.utils;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.apache.commons.lang3.StringEscapeUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import echoquery.sql.SingletonConnections;
+
+public class VisualizationUtil {
+  private static final Logger log =
+      LoggerFactory.getLogger(VisualizationUtil.class);
+  private static Connection conn = SingletonConnections.getStateInstance();
+
+  public static void updateDisplayText(String message) {
+    try {
+      Statement statement = conn.createStatement();
+      statement.executeUpdate(
+          "update sessions set display='" + StringEscapeUtils.escapeJava(message) + "'where id=1");
```

Comment:
right now it only works for one session and doesn't reset between sessions, but as a future PR I'll actually use session id here